hasher-matcher-actioner/hmalib/common/s3_adapters.py

```diff
@@ -33,6 +35,9 @@
 s3_client = boto3.client("s3")
 HashRowT = t.Tuple[str, t.Dict[str, t.Any]]
 
+# Signal types that s3_adapters should support
+KNOWN_SIGNAL_TYPES: t.Type[SignalType] = [VideoMD5Signal, PdqSignal]
```

Comment:
nit: why a module level const vs a static method of the `ThreatExchangeS3Adapter` class?